src/Compilers/Lua/Test/Portable/Lexical/LexicalTestData.cs

```diff
@@ -318,8 +338,8 @@ public static IEnumerable<ShortToken> GetTokens(LuaSyntaxOptions options)
 
             yield return new ShortToken(
                 SyntaxKind.HashStringLiteralToken,
-                $"`{shortStringContentText}`",
-                Hash.GetJenkinsOneAtATimeHashCode(shortStringContentValue.AsSpan()));
+                    $"`{shortStringContentText}`",
+                    Hash.GetJenkinsOneAtATimeHashCode(shortStringContentValue.AsSpan()));
```

Comment:
Extra indentation should be removed and the value passed to the hash method should be fixed:
```suggestion
            yield return new ShortToken(
                SyntaxKind.HashStringLiteralToken,
                $"`{shortStringContentText}`",
                Hash.GetJenkinsOneAtATimeHashCode(shortStringContentValue.ToLowerInvariant().AsSpan()));
```